README.md

```diff
@@ -15,9 +15,9 @@ Packages
 
 We provide many packages into the same project.
 
-### JSon Schema
+### JSON Schema
 
-[ETSI.org][3] proposal of implementation using the [JSON][2] language (instead of ASN.1 by default).
+[ETSI.org][3] proposal of implementation using the [JSON][2] language (instead of ASN.1 UPER by default).
```

Comment:
Semantically, those two changes about JSON schemas belong to another, separate commit.

Reply:
Well noted!